utils/build-script-impl

```diff
@@ -83,6 +83,9 @@ KNOWN_SETTINGS=(
     xctest-build-type           "Debug"          "the build variant for xctest"
     swiftpm-build-type          "Debug"          "the build variant for swiftpm"
     swiftsyntax-build-type      "Debug"          "the build variant for swiftSyntax"
+    # When this flag is set, the build-script will only build/install the libParser library.
+    # This is a walk-around of having a separate build product for swift-syntax parser
```

Comment:
typo `walk-around` -> `workaround`. Also good to add `temporary workaround`

Reply:
Done